### ISSUE TITLE
libcurl problem

### ISSUE THREAD

**Issue by nordsam** [nodkz/mongodb-memory-server#146]
When trying to run mongodb-memory-server in arch linux

I get this error
```
.../node_modules/.cache/mongodb-memory-server/mongodb-binaries/4.0.3/mongod: 
/usr/lib/libcurl.so.4.5.0: version `CURL_OPENSSL_3' not found 
(required by .../node_modules/.cache/mongodb-memory-server/mongodb-binaries/4.0.3/mongod)
```

`LD_PRELOAD=/usr/lib/libcurl-compat.so.4.5.0 npm test` and
`LD_PRELOAD=/usr/lib/libcurl-gnutls.so.4.5.0 npm test`
result to:
```
.../node_modules/.cache/mongodb-memory-server/mongodb-binaries/4.0.3/mongod: relocation error: 
.../node_modules/.cache/mongodb-memory-server/mongodb-binaries/4.0.3/mongod: 
symbol SSLv3_method version OPENSSL_1.0.0 not defined in file libssl.so.1.0.0 with link time reference
```

**Comment by nodkz** [comment]
I don't know how we can fix this issue automatically. Cause this is `mongod` binary requirement which we cannot track before starting the instance.

I think that we should provide a better error message for user with a some solution.

You may create PR with adding a couple of lines here
https://github.com/nodkz/mongodb-memory-server/blob/master/src/util/MongoInstance.ts#L213-L215

**Comment by hasezoey** [comment]
Closing in favor of #248
